_sass/navigation.scss

```diff
@@ -41,9 +42,12 @@ a.nav-text:hover {
 a.subnav-item:hover {
 	color: rgba(0, 179, 159, 1) !important;
 }
-
+a.subnav-item.active{
```

Comment:
I will be removing this before the code gets merged, this selector is of no use.